packages/cli/src/dbt/validation.ts

```diff
@@ -31,10 +31,6 @@ export const validateDbtModel = async (
     const validator = new ManifestValidator(manifestVersion);
     const results = models.reduce<DbtModelsGroupedByState>(
         (acc, model) => {
-            if (model.compiled === undefined) {
```

Comment:
@ZeRego do you remember what this was actually checking for?

Reply:
would skip models that weren't compiled. eg: `lightdash compile -s tag:lightdash`